### DISCUSSION TITLE
如何连接私有化部署的conan仓库

### DISCUSSION THREAD

**Discussion by Little-Cloud-Cloud** [xmake-io/xmake#5614] [General]
add_repositories("my-repo git@github.com:myrepo/xmake-repo.git") 这样的用法是否也使用conan的私有化仓库，我尝试了几次都没有成功

**Comment by waruqi** [comment]
不支持，走 conan 的配置，https://github.com/xmake-io/xmake/blob/cd0eb0da726f66fe3137e992db79c5a7d059c26c/xmake/modules/package/manager/conan/configurations.lua#L26

如何连接，翻下 conan 文档，我没试过
